hoot-core/src/main/cpp/hoot/core/io/Serializable.h

```diff
@@ -1,71 +0,0 @@
-/*
```

Comment:
Retired this. It was only used for serializing object during distributed Hadoop operations and an annoyance at this point. If we need to serialize object ever again, it can be revived. Most of the existing implementations would need to be selectively reworked as they all seem out of date.